miso-web/src/spring-test/java/uk/ac/bbsrc/tgac/miso/webapp/springtest/RunLibraryQcStatusRestControllerST.java

```diff
@@ -0,0 +1,89 @@
+package uk.ac.bbsrc.tgac.miso.webapp.springtest;
+
+import org.junit.Test;
+
+import uk.ac.bbsrc.tgac.miso.dto.Dtos;
+import org.springframework.security.test.context.support.WithMockUser;
+import static org.junit.Assert.*;
+import uk.ac.bbsrc.tgac.miso.core.data.RunLibraryQcStatus;
+import uk.ac.bbsrc.tgac.miso.dto.RunLibraryQcStatusDto;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+
+
+public class RunLibraryQcStatusRestControllerST extends AbstractST {
+
+  private static final String CONTROLLER_BASE = "/rest/runlibraryqcstatuses";
+  private static final Class<RunLibraryQcStatus> entityClass = RunLibraryQcStatus.class;
+
+  private List<RunLibraryQcStatusDto> makeCreateDtos() {
+    RunLibraryQcStatusDto dto1 = new RunLibraryQcStatusDto();
+    dto1.setQcPassed(true);
+    dto1.setDescription("one");
+
+
+
+    RunLibraryQcStatusDto dto2 = new RunLibraryQcStatusDto();
+    dto2.setQcPassed(false);
+    dto2.setDescription("two");
+
+    return Arrays.asList(dto1, dto2);
+  }
+
+  @Test
+  @WithMockUser(username = "admin", password = "admin", roles = {"INTERNAL", "ADMIN"})
+  public void testBulkCreateAsync() throws Exception {
+    // only admin can create
+    List<RunLibraryQcStatus> statuses = baseTestBulkCreateAsync(CONTROLLER_BASE, entityClass, makeCreateDtos());
```

Comment:
If you save makeCreateDtos() to a variable, you can then
`assertEquals(dtos.get(0).getDescription(), statuses.get(0).getDescription());`
clearer, and less prone to breaking